packages/x-tree-view/src/internals/plugins/useTreeViewItems/useTreeViewItems.ts

```diff
@@ -200,22 +191,21 @@ export const useTreeViewItems: TreeViewPlugin<UseTreeViewItemsSignature> = ({
     params.getItemLabel,
   ]);
 
-  const getItemsToRender = () => {
-    const getPropsFromItemId = ({
-      id,
-      children,
-    }: TreeViewItemIdAndChildren): ReturnType<typeof instance.getItemsToRender>[number] => {
+  const getItemsToRender = React.useCallback(() => {
```

Comment:
The `useCallback:` was not needed since we call this function on every render

Reply:
but you added it anyways? Am I misunderstanding something? 🤯

Reply:
I read the change backward, fixed :grimacing: